snapcraft.yaml

```diff
@@ -27,32 +27,28 @@ apps:
 parts:
   ubuntu-personal-store-qml:
     plugin: dump
-    source: https://github.com/snapcore/snapweb
-    source-type: git
+    source: .
     stage-packages:
+      - gksu
       - qmlscene
       - qml-module-qtquick2
       - qml-module-qtquick-controls
       - qml-module-ubuntu-web
     after: [desktop-qt5]
-    organize:
-      ubuntu-personal-store/ubuntu-personal-store.qml: ubuntu-personal-store.qml
-      ubuntu-personal-store/pkg: pkg
-      ubuntu-personal-store/setup: setup
     stage:
       - ubuntu-personal-store.qml
-      - pkg/*
+      - authenticate
```

Comment:
I forgot to stage authenticate. 
Anyway, that was not the reason why it was failing in Xenial